_data/projects/CPP.yml

```
@@ -0,0 +1,9 @@
+name: CPP
+desc: CPP repository contains all algorithms, resources related to competitive programming as well as for technical interviews.
+site: https://github.com/akshitagit/CPP
+tags:
+  - cpp
+  - programming
+upforgrabs:
+  name: good first issue
+  link: https://github.com/akshitagit/CPP/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

:art: let's use the simpler URL here

```suggestion
  link: https://github.com/akshitagit/CPP/labels/good%20first%20issue
```